packages/dappmanager/src/modules/eth2migration/getPrysmValidatorImage.ts

```diff
@@ -0,0 +1,48 @@
+import { imagesList } from "../docker/api";
+import { Eth2Network } from "./params";
+import semver from "semver";
+
+/**
+ * Fetch _SOME_ image from the available prysm package
+ * MUST be fetched dynamically here because we don't know when user will do the migration
+ * They may have an old version of Prysm or a newer version of Prysm.
+ * @param prysmOldDnpName
+ * ```
+ * validator.prysm.dnp.dappnode.eth:0.1.5
+ * ```
+ */
+export async function getPrysmValidatorImage({
+  prysmOldDnpName,
+  network
+}: {
+  prysmOldDnpName: string;
+  network: Eth2Network;
+}): Promise<string> {
+  // TODO: determine stable versions that support the prysm cli used in the eth2migration
+  const prysmStableVersion = network === "mainnet" ? "1.0.22" : "0.1.7";
```

Comment:
Don't assume there are only two networks, there may be more than mainnet and prater

Reply:
agree, now is retrieved from the starting function `getMigrationParams` depending on the network